fog/report/server/src/bin/main.rs

```diff
@@ -20,21 +20,25 @@ fn main() {
 
     let materials = Materials::try_from(&config).expect("Could not read cryptographic materials");
 
+    let database_url = env::var("DATABASE_URL").expect("DATABASE_URL environment variable missing");
     let db = SqlRecoveryDb::new_from_url(
-        &env::var("DATABASE_URL").expect("DATABASE_URL environment variable missing"),
+        &database_url,
         config.postgres_config.clone(),
         logger.clone(),
     )
-    .expect("Failed connecting to database");
+    .unwrap_or_else(|err| {
+        panic!(
+            "fog-report cannot connect to database '{}': {:?}",
+            database_url, err
+        )
+    });
 
     let mut server = Server::new(db, &config.client_listen_uri, materials, logger.clone());
     server.start();
 
-    let config2 = config.clone();
-    let get_config_json = Arc::new(move || {
-        serde_json::to_string(&config2)
-            .map_err(|err| RpcStatus::with_message(RpcStatusCode::INTERNAL, format!("{:?}", err)))
-    });
+    let config_json = serde_json::to_string(&config)
```

Comment:
Might as well panic here if we can't convert the config to json (we panic on other failures above).